src/components/Calculator.js

```diff
@@ -1,49 +1,70 @@
+
+  const initialValue = calcData.next || calcData.total || 0;
+  CalBtn.propTypes = {
+    incrementNumber: PropTypes.number,
+    characterStr: PropTypes.string,
+    bgColor: PropTypes.string.isRequired,
+  };
+
+  CalBtn.defaultProps = {
+    incrementNumber: 0,
+    characterStr: '',
+  };
 
-function Calculator() {
   return (
     <section className="container">
-      <input disabled value={0} className="show-input" />
+      <input type="text" className="show-input" value={initialValue} readOnly />
       <aside>
         <div>
-          <CalBtn incrementNumber="AC" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="+/-" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="%" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="+" bgColor="bg-orange btn p-normal" />
+          <CalBtn characterStr="AC" bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="+/-" bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="%" bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="+" bgColor="bg-orange btn p-normal" />
         </div>
         <div>
-          <CalBtn incrementNumber="7" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="8" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="9" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="x" bgColor="bg-orange btn p-normal" />
+          <CalBtn incrementNumber={7} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={8} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={9} bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="x" bgColor="bg-orange btn p-normal" />
         </div>
         <div>
-          <CalBtn incrementNumber="4" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="5" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="6" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="-" bgColor="bg-orange btn p-normal" />
+          <CalBtn incrementNumber={4} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={5} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={6} bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="-" bgColor="bg-orange btn p-normal" />
         </div>
         <div>
-          <CalBtn incrementNumber="1" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="2" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="3" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="+" bgColor="bg-orange btn p-normal" />
+          <CalBtn incrementNumber={1} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={2} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={3} bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="/" bgColor="bg-orange btn p-normal" />
```

Comment:
- [ ] This is the symbol `÷` used for division inside the operate.js file which should be the same symbol you must use on your `Calculator` component. Below is the error message I got while tring to test division sign on your calculator. Kindly fix it.

### Error
![division-error](https://user-images.githubusercontent.com/26597432/234982054-bda4594f-74e6-4193-9da4-d268a7849908.png)